swift-sdk/Internal/InAppManager.swift

```diff
@@ -532,7 +531,7 @@ class InAppManager: NSObject, IterableInternalInAppManagerProtocol {
     
     private var syncResult: Pending<Bool, Error>?
     private var lastSyncTime: Date?
-    private let moveToForegroundSyncInterval: Double = 1.0 * 60.0 // don't sync within sixty seconds
+    private var moveToForegroundSyncInterval: Double = 1.0 * 60.0 // don't sync within sixty seconds
```

Comment:
Was this change needed?

Reply:
Yes or the tests were failing when not being able to set a custom inapp interval